joss-paper/paper.md

```diff
@@ -0,0 +1,123 @@
+---
+title: '21cmFAST v3: A fully Python-integrated simulation code to generate 3D 21cm fluctuation fields through cosmic history.'
```

Comment:
maybe something like: 
21cmFAST v3: A Python-integrated C code for generating 3D realizations of the cosmic 21cm signal and its component fields
or maybe without component fields

Reply:
Yup, I like that better (without the component fields).